keyboards/ortho5by12/info.json

```
@@ -0,0 +1,137 @@
+{
+  "keyboard_name": "ortho5by12",
+  "keyboard_folder": "ortho5by12",
+  "url": "https://github.com/itsnoteasy/misc/blob/master/ortho5by12.zip",
+  "maintainer": "gipetto",
```

This should reference your GitHub username.

```suggestion
  "maintainer": "itsnoteasy",
```